commit 8b769ba87507a1f9df462188b5dc743c6dfac7c1
Author: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
Date:   Wed Feb 22 10:42:02 2017 +0530

    powerpc/mm/hash: Always clear UPRT and Host Radix bits when setting up CPU
    
    commit fda2d27db6eae5c2468f9e4657539b72bbc238bb upstream.
    
    We will set LPCR with correct value for radix during int. This make sure we
    start with a sanitized value of LPCR. In case of kexec, cpus can have LPCR
    value based on the previous translation mode we were running.
    
    Fixes: fe036a0605d60 ("powerpc/64/kexec: Fix MMU cleanup on radix")
    Acked-by: Michael Neuling <mikey@neuling.org>
    Signed-off-by: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1355_8b769ba87507a1f9df462188b5dc743c6dfac7c1.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1355_8b769ba87507a1f9df462188b5dc743c6dfac7c1.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
